force-app/main/default/triggers/OpportunityTriggers.trigger

```diff
@@ -0,0 +1,38 @@
+trigger OpportunityTriggers on Opportunity (before update, before delete) {
+    if (trigger.isBefore) {    
+        if (trigger.isUpdate) {
```

Comment:
You can combine these instead of making them nested. It depends on what you have after it, but if there is no else from it, I would combine them. 
`if (Trigger.isBefore && Trigger.isUpdate) {`